contracts/contracts/MessageProcessor.sol

```diff
@@ -67,6 +70,8 @@ contract MessageProcessor is Ownable, SnarkCommon, Hasher, CommonUtilities, IMes
     vkRegistry = IVkRegistry(_vkRegistry);
     poll = IPoll(_poll);
     mode = _mode;
+    batchHashes = poll.getBatchHashes();
+    currentBatchIndex = batchHashes.length;
```

Comment:
this will always be 1 at this time

Reply:
currentBatchIndex should be set to 1 directly, no need to read from `batchHashes.length;`

Reply:
@aleksandar-veljkovic this is not resolved, it's inefficient to read batchHashes.length when we know it's always 1 and can just pass 1

Reply:
sry i didn't push changes, we change locally and mark for us to know what we resolced. I didn't want to spam with pushes for every few comments. I will push now.